plone/namedfile/picture.py

```diff
@@ -86,13 +97,13 @@ def create_picture_tag(
                     scale_obj = scale_view.scale(fieldname, scale, pre=True)
                     scale_url = scale_obj.url
                 else:
-                    # obj = self.resolve_uid_url(src)
-                    # scale_view = obj.unrestrictedTraverse("@@images", None)
-                    # scale_obj = scale_view.scale(fieldname, scale, pre=True)
-                    # scale_url = scale_obj.url
                     scale_url = self.update_src_scale(src=src, scale=scale)
                 source_srcset.append(f"{scale_url} {scale_width}w")
-            source_tag = soup.new_tag("source", srcset=",\n".join(source_srcset))
+            if not sizes:
+                sizes = f"(min-width: 576px) {target_width}px, 98vw"
```

Comment:
Why 576?

Reply:
https://github.com/plone/plonetheme.barceloneta/blob/master/plonetheme/barceloneta/theme/css/barceloneta.css#L774
;)

Reply:
this fallback should be the same value as here https://github.com/plone/plone.base/pull/80/files#diff-935b88b2d094796541820b59f37748d9c86c288326d234e94b20de4ff09da231R1777